iocell.8

```diff
@@ -1187,17 +1187,63 @@ allow_sysvipc=0 | 1
 .nf
 .fam C
     A process within the jail has access to System V IPC
-    primitives.  In the current jail implementation, System V
-    primitives share a single namespace across the host and
-    jail environments, meaning that processes within a jail
-    would be able to communicate with (and potentially interfere
-    with) processes outside of the jail, and in other jails.
+    primitives. Prior to FreeBSD 11.0, System V primitives
+    share a single namespace across the host and jail
+    environments, meaning that processes within a jail would be
+    able to communicate with (and potentially interfere with)
+    processes outside of the jail, and in other jails.  In
+    FreeBSD 11.0 and later, this setting is deprecated in favor
+    of sysvmsg, sysvsem, and sysvshm.
 
     Default: 0
     Source: jail(8)
 
 .fam T
 .fi
+sysvmsg=disable | inherit | new
+.PP
+.nf
+.fam C
+    Allow access to SYSV IPC message primitives.  If set to
+    inherit, all IPC objects on the system are visible to this
+    jail, whether they were created by the jail itself, the base
+    system, or other jails.  If set to new, the jail will have
+    its own key namespace, and can only see the objects that it
+    has created; the system (or parent jail) has access to the
+    jail's objects, but not to its keys.  If set to disable, the
+    jail cannot perform any sysvmsg-related system calls.
+    Ignored in FreeBSD 10.3 and earlier.
+
+    Default: new
+    Source: jail(8)
+
+.fam T
+.fi
+sysvsem=disable | inherit | new
+.PP
+.nf
+.fam C
+    Allow access to SYSV IPC semaphore primitives in the same
+    manner as sysvmsg.  Ignored in FreeBSD 10.3 and earlier.
```

Comment:
One double space between line 1224 and 1227. Between "sysvmsg." and "Ignored"

BTW - iocell.8 seems full of double spaces. For example after every sentence ending with a point, I see a double space. If this should be cleaned up alltogether there should be a cleanup patch later on.